public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -290,7 +290,10 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
                   ) : detector.enabled &&
                     detector.curState === DETECTOR_STATE.INIT ? (
                     <EuiHealth color={DETECTOR_STATE_COLOR.INIT}>
-                      Initializing
+                      {detector.initProgress
+                        ? //@ts-ignore
+                          `Initializing (${detector.initProgress.percentageStr} complete)`
+                        : 'Initializing'}
```

Comment:
When would the progress bar be in initializing state but not `initProgress`?

Reply:
on unpatched backend.  Only latest backend has initProgress.